src/app/static/src/app/components/plots/MapAdjustScale.vue

```diff
@@ -101,48 +107,52 @@ import {ColourScaleType} from "../../store/colourScales/colourScales";
     export default Vue.extend<Data, Methods, Computed, Props>({
         name: "MapAdjustScale",
         props: {
+            name: String,
```

Comment:
I added this name property to allow the component to use it to construct a unique name for its radio button group -  without this, you could have both components open at once and acting as the same group (selecting in one component cleared selection in the other).